spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -300,14 +315,17 @@
           stub_request_passed_proofing_results
         end
 
-        it_behaves_like('enrollment with a status update', passed: true, status: 'passed')
+        it_behaves_like(
+          'enrollment with a status update', passed: true, status: 'passed',
+                                             response_json: UspsInPersonProofing::Mock::Fixtures.
+                                             request_passed_proofing_results_response
+        )
```

Comment:
I believe this style is more consistent with the rest of the codebase (and a little easier to read imo). I think I'll see if there's a rubocop rule to enforce this better in a separate PR

Similar comment for the other places below with the same styling

```suggestion
        it_behaves_like(
          'enrollment with a status update',
          passed: true,
          status: 'passed',
          response_json: UspsInPersonProofing::Mock::Fixtures.
            request_passed_proofing_results_response
        )
```